frontend/script.js

```diff
@@ -50,104 +36,213 @@ const init = () => {
+                            deleteTodo(element1.id) 
+                        }
+                    })
+                    
+                })  
+                             
+
+            }
+           
+        })
+
+    }) 
+})
+
+
 // Filter Todo and See if todo exists
 filterSearch.addEventListener('keypress', function(e){
     
 })
 
 
-// let body = {}
+async function fetchData() {
 
-async function fetchAPI() {
+    // await postData()
     
-
     let response = await fetch(`http://todo-app-keicee.herokuapp.com/api/todos/`, {
         "method": "GET",
-        // body: JSON.stringify({
-        //     id:"1", 
-        //     description: "Kelechi Egekenze"
-        // }),
         "headers": {
             "Content-Type": "application/json; charset=UTF-8"
         },
     })
 
     let data = await response.json()
     // deletePosts(data.data)
-    console.log(data.data);
+    // console.log(data.data);
     return data.data
 }
 
-fetchAPI()
+fetchData()
 
 
-async function postData(){
-        let response = await fetch(`http://todo-app-keicee.herokuapp.com/api/todos/`, {
-            "method": "POST",
-            body: JSON.stringify({ 
-                description: input.value
-            }),
-            "headers": {
-                "Content-Type": "application/json; charset=UTF-8"
-            },
-        })
-    
-        let data = await response.json()
-        // deletePosts(data.data)
-        console.log(data);
+async function postData() {
+
+    let response = await fetch(`http://todo-app-keicee.herokuapp.com/api/todos/`, {
+        "method": "POST",
+        body: JSON.stringify({
+            description: input.value
+        }),
+        "headers": {
+            "Content-Type": "application/json; charset=UTF-8"
+        },
+    })
+
+    let data = await response.json()
+    // deletePosts(data.data)
+    console.log(data);
+
+}
+
+async function updateTodo(id){
+
+    let response = await fetch(`http://todo-app-keicee.herokuapp.com/api/todos/${id}`, {
+        "method": "PUT",
+        body: JSON.stringify({
+            description: 'Egekenze Kelechi'
+        }),
+        "headers": {
+            "Content-Type": "application/json; charset=UTF-8"
+        },
+    })
+
+    let data = await response.json()
+    // deletePosts(data.data)
+    console.log(data);
 }
 
-// async function deletePosts(data){
-//     let reply 
 
-//     data.forEach(element => {
-//         reply = element.id
-//         console.log(reply);
-//     });
 
-//     console.log(reply);
-    
-//     let response = await fetch(`http://todo-app-keicee.herokuapp.com/api/todos/${reply}`, {
-//         "method": "DELETE",
-//         "headers": {
-//             "Content-Type": "application/json; charset=UTF-8"
-//         },
-        
-//     })
-    
+// Adds complete = true for checked Todos.
+async function addCompletedTodo(id, newBody){
```

Comment:
Why exactly are you passing this newBody variable to this endpoint, does the endpoint even need this? Does it request for a "complete" boolean??